Gemfile

```diff
@@ -1,6 +1,4 @@
-require File.expand_path('Gemfile.global.rb', File.dirname(__FILE__))
-
-MiqBundler.include_gemfile("gems/pending/Gemfile", binding)
+eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
```

Comment:
https://github.com/bundler/bundler/blob/master/lib/bundler/dsl.rb#L32 (I saw it in there since at least 1.2.0 - didn't look back any further)

I guess I could add it to the commit message...seems pretty clear what it does though.